script/bootstrap

```diff
@@ -83,6 +83,9 @@ function bootstrap() {
   if (process.env.JANKY_SHA1 && process.platform === 'win32')
     apmInstallCommand += ' --arch=ia32';
 
+  if (!process.env.ATOM_NODE_VERSION)
+    process.env.ATOM_NODE_VERSION = require('../package.json').electronVersion;
```

Comment:
This shouldn't be needed anymore right since `apm` reads `electronVersion`?

If we still want it as a fallback, it should probably be renamed to the new `ATOM_ELECTRON_VERSION` env var support in apm.